deploy.sh

```diff
@@ -2,14 +2,19 @@
 
 export BUILD_TARGET=$1
 if [ -z "$BUILD_TARGET" ]; then
-    echo "USAGE: ./deploy.sh <target>   (base, app, etc.)" >&2
+    echo "USAGE: ./deploy.sh <target> [options]  (base, app, etc.)" >&2
+    echo " --test -- (for legacy deployment only) only deploy to ami-test, not all workers"
+    ehco " --only='..' -- only build these packer builders"
```

Comment:
s/`ehco`/`echo`
Also would be nice if the help message provided the options, and how should we pass them (space or comma separated?)

Reply:
Good point.  I was thinking of actually switching this to something like `--cloud=aws` to only build specific clouds, and then translating that to packer builders later.